idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/RemoveModifierFix.kt

```diff
@@ -137,5 +138,23 @@ class RemoveModifierFix(
                 listOf(RemoveModifierFix(funInterface, KtTokens.FUN_KEYWORD, isRedundant = false))
             }
         }
+
+        fun getElementName(modifierListOwner: KtModifierListOwner): String {
```

Comment:
Why did you have to move this from `AddModifierFix`?

Reply:
Since AddModifierFix can depend on RemoveModifierFix, but not the other way around.

Reply:
Ah, forgot that you were keeping `AddModifierFix` in `idea` module.